src/pages/AlumniPage.jsx

```diff
@@ -1,90 +1,179 @@
+                    <img src={alum_vector2} className="w-[80%] h-[130%] absolute transform translate-x-[25%] translate-y-[-53%] z-5 "/>
                 </div>
             </div>
 
-            <div className="max-w-[1264px] w-full flex items-center justify-center h-[385px] mt-20 bg-gray-200 rounded-lg p-8">
-                <div className="max-w-[1160px] w-full">
-                    <h2 className="text-black font-inter text-center text-4xl font-medium tracking-wide">
-                        CARE TO REACH OUT? CONNECT WITH US!
-                    </h2>
-                    <div className="flex items-center justify-center gap-10 mt-6">
-                        <a href="mailto:fusion.uci.networking@gmail.com" target="_blank" rel="noopener noreferrer">
-                            <img src={Gmail} alt="Gmail Icon" className="w-[127.76px] h-[97.26px] opacity-1"/>
+            <div className="w-full h-[100px] bg-[#fff5eb]">
+
+            </div>
+
+            {/* connect */}
+            <div className="bg-white w-full flex flex-col items-center justify-start">
+                <div style={{ 
+                        backgroundImage: `url(${alum_picnic2})`, 
+                        backgroundColor: '#fff5eb', 
+                        backgroundBlendMode: 'overlay' 
+                    }} 
+                    className="w-full h-[752px] bg-cover bg-center flex items-center justify-center"
+                    >
+                    <div className="w-[42%] h-[300px] bg-white flex flex-col justify-start items-center px-12 py-10 rounded-tr-3xl rounded-bl-3xl border-2 border-black">
+                        <h1 className="font-[700] text-lg text-[#FFA523] tracking-[.1em]">INTERESTED?</h1>
+                        <h2 className="font-[600] text-4xl mt-3 text-[#FFA523]">Stay connected to learn more.</h2>
+                        <div className="w-[60%] h-1/3 flex flex-row mt-12 border-2 border-black">
+                        <a href="fusion.uci.networking@gmail.com">
+                            <img src={alum_email}/>
                         </a>
-                        <a href="https://www.facebook.com/fusionuci/" target="_blank" rel="noopener noreferrer">
-                            <img src={Facebook} alt="Facebook Icon" className="w-[108.8px] h-[108.8px] opacity-1"/>
+                        <a href="https://www.instagram.com/fusionatuci/" className="ml-9">
+                            <img src={alum_facebook}/>
                         </a>
-                        <a href="https://www.linkedin.com/in/fusionatuci/" target="_blank" rel="noopener noreferrer">
-                            <img src={LinkedIn} alt="LinkedIn Icon" className="w-[108.15px] h-[108.8px] opacity-1"/>
+                        <a href="linkedin.com/in/fusionatuci" className="ml-9">
+                            <img src={alum_linkedin}/>
                         </a>
+                        </div>
                     </div>
                 </div>
             </div>
```

Comment:
All the icons and the container that these elements are in have a black outline that need to be removed